src/components/accordion-item/accordion-item.scss

```diff
@@ -155,33 +157,44 @@
 :host(:active),
 :host([active]),
 :host([expanded]) {
-  & .accordion-item-heading {
+  & .heading {
```

Comment:
For next time, please keep refactoring separate. It widens the scope of changes and review.